project-gallery.md

```diff
@@ -1,11 +0,0 @@
----
-layout: projects
-title: Project Gallery
-permalink: /projects/
-stylesheets:
-  - /css/projects.css
```

Comment:
you can also remove projects.css and projects.yml

Reply:
tbh just grep for projects and see what comes up

Reply:
should be good now!